app/Http/Controllers/config/laratrust_seeder.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+return [
+    'role_structure' => [
+        'superadministrator' => [
+            'users' => 'c,r,u,d',
+            'acl' => 'c,r,u,d',
+            'profile' => 'r,u',
+        ],
+        'administrator' => [
+            'users' => 'c,r,u,d',
+            'profile' => 'r,u',
+        ],
+        'editor' => [
+            'profile' => 'r,u',
+        ],
+        'author' => [
+            'profile' => 'r,u',
+        ],
+        'contributor' => [
+            'profile' => 'r,u',
+        ],
+        'supporter' => [
+            'profile' => 'r,u',
+        ],
+        'subscriber' => [
+            'profile' => 'r,u',
+        ],
+    ],
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/config/laratrust_seeder.php</br>**Line**: 31</br>Each line in an array declaration must end in a comma